src/edu/csus/ecs/pc2/ui/team/QuickSubmitter.java

```diff
@@ -91,6 +93,25 @@ public List<File> getAllCDPsubmissionFileNames(IInternalContest mycontest, Strin
 
         return files;
     }
+    
+    
+    public File sendSubmission(File sourceFile) throws Exception {
+        File submittedFile = null;
+
+        Language language = LanguageUtilities.guessLanguage(getContest(), sourceFile.getAbsolutePath());
+        if (language == null) {
+            String ext = LanguageUtilities.getExtension(sourceFile.getAbsolutePath());
+            throw new Exception("Cannot identify language for ext= " + ext + " = Can not send submission for file " + sourceFile.getAbsolutePath());
+        } else {
+            Problem problem = guessProblem(getContest(), sourceFile.getAbsolutePath());
+            controller.submitJudgeRun(problem, language, sourceFile.getAbsolutePath(), null);
+            System.out.println("submitted run send with language: " + language + " and problem: " + problem.getShortName() + " title:" + problem + " as " + getContest().getClientId());
```

Comment:
I thought we agreed that classes like this shouldn't be writing to stdout.  The log is available (it's set when `QuickSubmitter.setContestAndController()` is called).  I think this message should be written to the _log_, and only output to stdout _if the log is null_.